context/fixtures/food/output-columns.txt

```diff
@@ -0,0 +1,8 @@
+fruit.csv,green.tsv,sweet.txt,vegetable.txt
+Apple,Apple,Apple,Broccoli
+Lemon,Broccoli,Kiwi,Carrot
+Lime,Cucumber,Pe,Pepper
```

Comment:
What the????

Reply:
@scottx611x : Dumb, but fairly interesting bug. It's now using dataframer, and there was an input fixture where I had not added a header row, and the first item was "banana"... guess what it does when you ask it to guess the delimiter?

Pushed a fix, and if it passes, I'll merge and tag the release.